oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConstants.java

```diff
@@ -27,39 +27,56 @@ private OracleConstants() {
   }
 
   public static final String PLUGIN_NAME = "Oracle";
-  public static final String ORACLE_CONNECTION_STRING_SID_FORMAT = "jdbc:oracle:thin:@%s:%s:%s";
-  public static final String ORACLE_CONNECTION_STRING_SERVICE_NAME_FORMAT = "jdbc:oracle:thin:@//%s:%s/%s";
+  // Updating connection strings to accept protocol (e.g., jdbc:oracle:thin:@<protocol>://<host>:<port>/<SID>)
+  public static final String ORACLE_CONNECTION_STRING_SID_FORMAT = "jdbc:oracle:thin:@%s:%s:%s/%s";
```

Comment:
We can add a constant `ORACLE_CONNECTION_STRING_SID_FORMAT_WITH_PROTOCOL` instead of modifying the older one and use the new one when `sslMode` is set to `true` otherwise use the old connection string. 

This would make the change backward compatible and ensure CDAP connectors are not affected.

Reply:
got it, updated [b31e53c08a5d67793ec914efcfefe270c5e3a434]